src/plugins/auto/src/auto_compiled_model.cpp

```diff
@@ -63,6 +63,12 @@ ov::Any AutoCompiledModel::get_property(const std::string& name) const {
         supported_properties.insert(supported_properties.end(), ro_properties.begin(), ro_properties.end());
         supported_properties.insert(supported_properties.end(), rw_properties.begin(), rw_properties.end());
         return decltype(ov::supported_properties)::value_type(supported_properties);
+    } else if (name == ov::enable_profiling) {
+        auto rw_properties = default_rw_properties();
```

Comment:
m_need_perf_counters in m_context is the variable which store the value of ov::enable_profiling property.